packages/ts/file-router/src/vite-plugin/createRoutesFromMeta.ts

```diff
@@ -76,11 +76,19 @@ export default function createRoutesFromMeta(views: RouteMeta, { code: codeFile
       ([statement]) => statement as ts.ImportDeclaration,
     ),
   ];
+  const errors: string[] = [];
   let id = 0;
 
   const routes = transformTreeSync<RouteMeta, CallExpression>(
     views,
-    (view) => view.children.values(),
+    (view) => {
+      const paths = view.children.map((c) => c.path);
+      const uniquePaths = new Set(paths);
+      paths
+        .filter((p) => !uniquePaths.delete(p))
+        .forEach((p) => errors.push(`console.error("Two views share the same path: ${p}");`));
```

Comment:
Would it possible to change the error message to include the name of all the views (not just "Two") sharing the same path?

Reply:
To my understanding, the concerned views cannot be more than two. And for the full path, the current object has no information about parents, so the only thing we can easily access is the current path item (e.g. "about").